Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.UWP/MapRenderer.cs

```diff
@@ -341,6 +349,10 @@ void UpdateHasZoomEnabled()
         void UpdateHasScrollEnabled()
         {
         }
+
+        void UpdateHasRotationEnabled()
```

Comment:
may as well keep it there. It doesn't hurt anything. If not can someone take it out? Or do I have to take it out?